frontend/packages/vsphere-plugin/src/resources/model.ts

```diff
@@ -0,0 +1,14 @@
+import { K8sModel } from '@console/dynamic-plugin-sdk';
+
+// /apis/operator.openshift.io/v1/kubecontrollermanagers/cluster'
+export const KubeControllerManagerModel: K8sModel = {
```

Comment:
you should use `useK8sModel` instead of this static model definition

Reply:
done